media_kit_video_controls/lib/src/controls/material.dart

```diff
@@ -0,0 +1,1604 @@
+/// This file is a part of media_kit (https://github.com/alexmercerind/media_kit).
```

Comment:
Perhaps split this huge file into several class files, which will improve code diffing in future PRs. Same for desktop.

Reply:
Definitely.